README.md

```diff
@@ -0,0 +1,81 @@
+# TDEI python lib OSW validation package
+
+This package validates the OSW geojson file. Package requires a OSW zip file path
+
+## System requirements
+
+| Software | Version |
+|----------|---------|
+| Python   | 3.10.x  |
+
+## What this package does?
+
+- It unzip the provided zip files
+- Check for nodes, edges and points geojson files inside the unzipped folder
+- Validate each file against schema, schema can be found here
+- Return true or false according to validation
+- you can check the error if it returned false.
+
+## Starting a new project with template
+
+- Add `python-lib-osw-validation` package as dependency in your `requirements.txt`
+- or `pip install python-lib-osw-validation`
```

Comment:
Does this work when the library is hosted in `testpy`?

Reply:
right now no, but once it is deployed to pypi then it would work